server/main_server/src/main_server_control_plane.cpp

```diff
@@ -547,7 +547,8 @@ namespace irods
     void server_control_executor::operator()()
     {
         signal( SIGINT, ctrl_plane_handle_shutdown_signal );
-        signal( SIGHUP, ctrl_plane_handle_shutdown_signal );
+        // SIGHUP is the signal to reload the configuration. This shouldn't interact with it
+        signal(SIGHUP, SIG_IGN);
```

Comment:
How did you arrive as this being the solution?

Reply:
The main server kept on having trouble with the control plane being terminated.

The control plane forwards the termination right back to the main server as it exits.